experiments/config/gmm_v0.yaml

```diff
@@ -26,7 +26,7 @@ fab:
       tune_step_size: true
     metropolis:
       n_outer_steps: 1
-      init_step_size: 5.
+      init_step_size: 10.
```

Comment:
Change so that behavior is exactly the same as before this issue (step size was being (incorrectly) multiplied by dim.